contracts/src/v0.8/ccip/pools/GHO/UpgradeableBurnMintTokenPool.sol

```diff
@@ -84,6 +84,18 @@ contract UpgradeableBurnMintTokenPool is Initializable, UpgradeableBurnMintToken
     _setRateLimitConfig(remoteChainSelector, outboundConfig, inboundConfig);
   }
 
+  /// @notice This function allows the owner to burn `amount` of the pool's token. This is
+  /// expected to be called while migrating liquidity to another pool and offboarding this
+  /// facilitator.
+  /// @dev New token pool should mint and transfer liquidity to this pool (since it does not
+  /// hold tokens any point in point, only mints/burns) which can be burnt and hence will reset
```

Comment:
typo: "hold tokens any point in point" -> probably meant "hold tokens at any point in time"

Reply:
[9a2bcd4](https://github.com/aave/ccip/pull/21/commits/9a2bcd4eb5317cb6f9d6f2ab240db844416d7afa)